app/services/participants/defer/validate_and_change_state.rb

```diff
@@ -6,11 +6,19 @@ module ValidateAndChangeState
       extend ActiveSupport::Concern
       include ActiveModel::Validations
 
+      REASONS = %w[
+        bereavement
+        long-term-sickness
+        parental-leave
+        career-break
+        other
+      ].freeze
```

Comment:
Are the deferral reasons the same for ecf and npq?

Reply:
yes